### PR TITLE
extend model options documentation for properties 'underscored' and 'underscoredAll'

### PR DESCRIPTION
**Pull request by Alexsey**
Better wording for [this](https://github.com/sequelize/sequelize/issues/7506) problem

### CODE REVIEW

File: lib/model.js [hunk 1]
```diff
@@ -670,8 +670,8 @@ class Model {
    * @param {Boolean}                 [options.omitNull] Don't persist null values. This means that all columns with null values will not be saved
    * @param {Boolean}                 [options.timestamps=true] Adds createdAt and updatedAt timestamps to the model.
    * @param {Boolean}                 [options.paranoid=false] Calling `destroy` will not delete the model, but instead set a `deletedAt` timestamp if this is true. Needs `timestamps=true` to work
-   * @param {Boolean}                 [options.underscored=false] Converts all camelCased columns to underscored if true
-   * @param {Boolean}                 [options.underscoredAll=false] Converts camelCased model names to underscored table names if true
+   * @param {Boolean}                 [options.underscored=false] Converts all camelCased columns to underscored if true. Will not convert column name if it set explicitly through options
+   * @param {Boolean}                 [options.underscoredAll=false] Converts camelCased model names to underscored table names if true. Will not convert names for models with option `freezeTableName` set to `true`
```
Comment (sushantdhiman) [inline lib/model.js]: Will not change model name if `freezeTableName` is set to `true`

File: lib/model.js [hunk 2]
```diff
@@ -670,8 +670,8 @@ class Model {
    * @param {Boolean}                 [options.omitNull] Don't persist null values. This means that all columns with null values will not be saved
    * @param {Boolean}                 [options.timestamps=true] Adds createdAt and updatedAt timestamps to the model.
    * @param {Boolean}                 [options.paranoid=false] Calling `destroy` will not delete the model, but instead set a `deletedAt` timestamp if this is true. Needs `timestamps=true` to work
-   * @param {Boolean}                 [options.underscored=false] Converts all camelCased columns to underscored if true
-   * @param {Boolean}                 [options.underscoredAll=false] Converts camelCased model names to underscored table names if true
+   * @param {Boolean}                 [options.underscored=false] Converts all camelCased columns to underscored if true. Will not convert column name if it set explicitly through options
```
Comment (sushantdhiman) [inline lib/model.js]: I think you meant `field` option, proper wording to relay this message would be,

"unless column name is explicitly set with `field` options"

Comment (Alexsey) [inline lib/model.js]: I meant options like `createdAt`. What do you mean under `field`? I have searched over [API](http://docs.sequelizejs.com/en/latest/api/sequelize/) page and didn't find `options.field` but only `options.fieldMap`

Comment (sushantdhiman) [inline lib/model.js]: `field` is an attribute option which can override column name. Used something like this

```js
orgName: {
   type: Sequelize.STRING,
   field: 'org_name'
}
```

You are talking about timestamp fields, you can change message to something like

"Will not affect timestamp fields named explicitly by model options"

**Comment by Alexsey** [comment]
If there is anything else you would like me to change in this PR to accept it - I'm open :)

**Pull request merged**